docs/src/modules/sandbox/Dependencies.ts

```diff
@@ -109,8 +110,7 @@ export default function SandboxDependencies(
 
     // TODO: consider if this configuration could be injected in a "cleaner" way.
     if (muiDocConfig) {
-      const muiCommitRef = process.env.PULL_REQUEST_ID ? process.env.COMMIT_REF : undefined;
-      versions = muiDocConfig.csbGetVersions(versions, { muiCommitRef });
+      versions = muiDocConfig.csbGetVersions(versions, { muiCommitRef: commitRef });
```

Comment:
This solves a bug on other repositories using docs-infra, e.g. https://deploy-preview-11848--material-ui-x.netlify.app/x/react-data-grid/editing/#full-featured-crud StackBlitz is broken, CodeSandbox works, this is because of https://github.com/stackblitz/core/issues/437